package.json

```diff
@@ -17,28 +17,28 @@
   },
   "private": true,
   "devDependencies": {
```

Comment:
@billti  I wonder if we should just be pinning these devDependencies to `latest`, instead of specific major versions. I ended up running `npm install foo@latest` on everything anyway, and we have a lockfile, so the version spec in `package.json` ends up being meaningless. (The exception would be the node.js typings, which we update along with node.js.) Thoughts?

Reply:
That seems a little dangerous. I know eslint has some quite significant changes, and I though we stayed on an older version of marked for a reason (I see it went from 4.x to 9.x now - we'll def. want to check the rendering is all looking good still).

What benefit would making all 'latest' construe?

Reply:
If all of these said `"latest"`, then instead of running the script I pasted in the PR description, I could just run `npm update` next time.

> I see it went from 4.x to 9.x now - we'll def. want to check the rendering is all looking good still

Yes, exactly, the bulk of my time was spent doing this validation. And also writing the PR description.

Considering that either way, I'm running `npm install foo@latest` on all the dependencies, and validating if something broke, what's the harm in just putting `latest` in package.json? If a major upgrade caused severe issues and we didn't want to take it, we would just pin to a downlevel version.

Reply:
OK. We can give it a shot here if you like, or just try it out as a follow-up.

Reply:
All these updates are already making me a bit nervous, I think I'd rather get this in and tackle that later! I just wanted to bring it up for discussion - thanks for thinking through it with me.